tests/api/test_file_filter.py

```diff
@@ -33,3 +33,71 @@
     assert await ff.match("spam.txt")
     assert await ff.match("ham")
     assert not await ff.match("ham.txt")
+    assert await ff.match("dir/spam.txt")
+    assert not await ff.match("dir/ham.txt")
+    assert await ff.match("dir.txt/spam")
```

Comment:
There is a difference with `.gitignore` for this case. Humm, I thought it is not significant, but now I think that it may be important.

Reply:
Well, it is not important in the context of using filters, because we handle files recursively and omit the content of a directory if its name is filtered out.

Reply:
I think this is fine, as it is specific to filter, not translate.